src/vmspawn/vmspawn.c

```diff
@@ -67,8 +76,12 @@ static int help(void) {
                "     --qemu-smp=SMP         Configure guest's SMP settings\n"
                "     --qemu-mem=MEM         Configure guest's RAM size\n"
                "     --qemu-kvm=            Configure whether to use KVM or not\n"
+               "     --qemu-vsock=          Configure whether to use qemu with a vsock or not\n"
```

Comment:
maybe write this as:

```
--qemu-vsock=BOOL
```

just as a hint to the user

Reply:
I will do the same for qemu-kvm as well.

Reply:
Just realised I forgot to update --secboot to --secure-boot in the man page so I will include that in this PR.

Reply:
Fixed in ef3bb39e80.